testing/testcafe/tests/editors/tagBox/label.ts

```diff
@@ -39,5 +44,50 @@ themes.forEach((theme) => {
         multiline: true,
       }, true, '#otherContainer');
     });
+
+    labelMods.forEach((labelMode) => {
+      test(`Label shouldn't be cutted for dxTagBox ${theme} in stylingMode=${stylingMode}, labelMode=${labelMode} (T1104913)`, async (t) => {
+        const tagBox = new TagBox('#container');
+
+        await t.click(tagBox.element);
+
+        await t
+          .expect(await compareScreenshot(t, `label-tag-box-styleMode=${stylingMode},labelMode=${labelMode},theme=${theme.replace(/\./g, '-')}.png`))
```

Comment:
Lee's move etalon name to a variable to reduce copypaste (55 and 62 row)?

Reply:
[commit](https://github.com/DevExpress/DevExtreme/pull/22471/commits/eb90394b50ac98160b9f9b8a15c21453b70813d6)